src/components/AccessibilityPolicyForm/index.jsx

```diff
@@ -106,23 +114,35 @@ export class AccessibilityPolicyForm extends React.Component {
       status = {
         alertType: 'danger',
         alertDialog: (
-          <div>
-            We are currently experiencing high volume. Try again later today or
-             send an email message to <a href={`mailto:${accessibilityEmail}`}>{accessibilityEmail}</a>.
-          </div>
+          <WrappedMessage
+            message={messages.accessibilityPolicyFormErrorHighVolume}
+            tagName="div"
+            values={{
+              emailLink: <a href={`mailto:${accessibilityEmail}`}>{accessibilityEmail}</a>,
+            }}
+          />
         ),
       };
     } else if (accessibilityStatus.type ===
         accessibilityActions.ACCESSIBILITY_FORM_SUBMIT_SUCCESS) {
+      const start = new Date('Mon Jan 29 2018 13:00:00 GMT (UTC)');
+      const end = new Date('Fri Feb 2 2018 21:00:00 GMT (UTC)');
```

Comment:
Is there any way to make these more dynamic?

Reply:
Yeah, I felt bad about how this ended up. What's the best way to create a `Date` that represents "a Monday" or "1300 UTC on any particular day"?